modules/summary_pipeline.py

```diff
@@ -12,7 +12,7 @@ def __init__(self, summary_options, standard_options, label=None, **kwargs):
     def expand(self, pages):
         home_pages = pages | "FilterHomePages" >> beam.Filter(utils.is_home_page)
 
-        _ = (
+        _ = (  # pragma: no branch
```

Comment:
What does this do?

Reply:
`# pragma: no branch` is used to tell the code coverage library to ignore this as a "partial branch" since it doesn't actually branch.

https://coverage.readthedocs.io/en/6.4.4/branch.html#structurally-partial-branches